CMakeLists.txt

```diff
@@ -4,17 +4,29 @@ cmake_minimum_required(VERSION 3.16)
 # If commented, the latest supported standard for your compiler is automatically set.
 # set(CMAKE_CXX_STANDARD 20)
 
-# Add project_options v0.26.2
+
+# Add project_options
 # https://github.com/aminya/project_options
-# Change the version in the following URL to update the package (watch the releases of the repository for future updates)
-include(FetchContent)
-FetchContent_Declare(_project_options URL https://github.com/aminya/project_options/archive/refs/tags/v0.26.2.zip)
-FetchContent_MakeAvailable(_project_options)
-include(${_project_options_SOURCE_DIR}/Index.cmake)
+# Change the version to update the package (watch the releases of the repository for future updates)
+include(cmake/CPM.cmake)
+CPMAddPackage(
+  NAME              project_options
+  GITHUB_REPOSITORY aminya/project_options
+  #VERSION           0.26.3
+  GIT_TAG           main
```

Comment:
Change this to a project_options version with `enable_cross_compiler()` ... _(Used this for testing)_

Reply:
I prefer using `FetchContent`, which is built-in